helm/cert-operator/templates/configmap.yaml

```diff
@@ -13,7 +13,7 @@ data:
         address: 'http://0.0.0.0:8000'
     service:
       crd:
-        labelSelector: '{{ .Values.Installation.V1.GiantSwarm.CertOperator.CRD.LabelSelector }}'
+        labelSelector: ''
```

Comment:
I've never interacted with this value before -- what does it do? Or at least, could the change affect which CRs this version of `cert-operator` will reconcile?

Reply:
It is empty everywhere. AFAIK that's some legacy unused stuff. But I'll make it configurable in case we need to change it